vaccine_feed_ingest/runners/ky/govstatus/normalize.py

```diff
@@ -0,0 +1,153 @@
+    latitude = site["lat"]
+    longitude = site["long"]
+    if latitude == "" or longitude == "":
+        return None
+    return schema.LatLng(
+        latitude=float(latitude),
+        longitude=float(longitude),
+    )
+
+
+def _get_contacts(site: dict):
+    ret = []
+
+    if "register_phone" in site:
+        raw_phone = site["register_phone"]
+        if raw_phone != "":
+            raw_phone = raw_phone.lstrip("tel:")
+            raw_phone = raw_phone.lstrip(" ")
+            raw_phone = raw_phone.lstrip("1")
+            raw_phone = raw_phone.lstrip("-")
+            raw_phone = raw_phone.lstrip(" ")
+            if raw_phone[3] == "-" or raw_phone[7] == "-":
+                phone = (
+                    "(" + raw_phone[0:3] + ") " + raw_phone[4:7] + "-" + raw_phone[8:12]
+                )
+                phone_notes = raw_phone[12:]
+            elif len(raw_phone) == 10:
+                phone = (
+                    "(" + raw_phone[0:3] + ") " + raw_phone[3:6] + "-" + raw_phone[6:10]
+                )
+                phone_notes = ""
+            else:
+                phone = raw_phone[0:14]
+                phone_notes = raw_phone[14:]
+
+            if phone_notes == "":
+                ret.append(schema.Contact(phone=phone, contact_type="booking"))
+            else:
+                phone_notes = phone_notes.lstrip(",")
+                phone_notes = phone_notes.lstrip(";")
+                phone_notes = phone_notes.lstrip(" ")
+                ret.append(
+                    schema.Contact(
+                        phone=phone,
+                        other=f"phone_notes:{phone_notes}",
+                        contact_type="booking",
+                    )
+                )
+
+    if "register_online_url" in site:
+        website = site["register_online_url"]
+        if website != "":
+            ret.append(schema.Contact(website=website, scontact_type="booking"))
+    return ret
+
+
+def _get_organization(site: dict):
+    # organization_name = ""
+    #
+    # if site["Organization Name"] == "":
+    #     return None
+
+    if _get_name(site)[0:6] == "Kroger":
+        return schema.Organization(name="Kroger", id="kroger")
+    if _get_name(site)[0:9] == "Walgreens":
+        return schema.Organization(name="Walgreens", id="walmart")
+    if _get_name(site)[0:7] == "Walmart":
+        return schema.Organization(name="Walmart", id="walmart")
+
+    # return schema.Organization(name=site["Organization Name"])
+
+
+def _get_notes(site: dict):
+    return []
+
+
+def _get_source(site: dict, timestamp: str) -> schema.Source:
+    return schema.Source(
+        data=site,
+        fetched_at=timestamp,
+        fetched_from_uri="https://govstatus.egov.com/kentucky-vaccine-map",
+        id=_get_id(site),
+        source=_source_name,
+    )
+
+
+def normalize(site: dict, timestamp: str) -> str:
+    normalized = schema.NormalizedLocation(
+        id=(_source_name + ":" + _get_id(site)),
```

Comment:
minor: slightly more standard way is to use f-string `f"{_source_name}:{_get_id(site)}"`

Reply:
Sure, applied.